src/main/java/com/rakovets/course/java/core/practice/looping_statements/Task10.java

```diff
@@ -40,6 +40,18 @@ public static void main(String[] args) {
     static int countPrimeNumber(int startNumber, int finishNumber) {
         //TODO
         // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return 0;
+        int countPrimeNumber = 0;
+        for(int i = startNumber; i <= finishNumber; i++) {
+            if(i <= 1){
```

Comment:
"Violation: There isn't separating white spaces (OR line break) for curly brake
Solution: Add white spaces"